spec/features/ctc/eitc_spec.rb

```diff
@@ -6,7 +6,6 @@
   before do
     allow_any_instance_of(Routes::CtcDomain).to receive(:matches?).and_return(true)
     Flipper.enable :eitc
-    Flipper.enable :puerto_rico_home_location
```

Comment:
Does calling Flipper.enable like this keep the feature flag turned on for later tests? I've been looking for documentation on whether this will automatically disable in a test context afterward...

Reply:
https://github.com/codeforamerica/vita-min/blob/main/spec/rails_helper.rb#L138 makes it so we start with a fresh (empty) set of flags every test

Reply:
nice, good to know. thanks.